### DISCUSSION TITLE
The audio contains Chinese and English，Can it be recognized at the same time?

### DISCUSSION THREAD

**Discussion by mack007liu** [openai/whisper#807] [General]
The audio contains Chinese and English, one man speaks Chinese, and the other woman translates into English. Can it be recognized at the same time?

**Comment by otakutyrant** [comment]
In order to transcribe a video that is primarily in Chinese but occasionally contains English. I tried `--language en,zh` but whisper complained that the argument was wrong. `--language en` caused all lines became `music`. `--language zh` worked very well that it could transcribe Chinese and English.

**Comment by phineas-pta** [comment]
no, whisper doesn't support audio with multiple languages
